tests/dbt/test_graph.py

```diff
@@ -665,11 +665,35 @@ def test_tag_selected_node_test_exist():
     assert len(dbt_graph.filtered_nodes) > 0
 
     for _, node in dbt_graph.filtered_nodes.items():
-        assert node.tags == ["test_tag"]
+        assert "test_tag" in node.tags
```

Comment:
It would be great if we could improve the way we're doing this selection without introducing a breaking change.

Reply:
I think this line change is the right test to make, your test should only be asserting that the tag exists, because that is a sufficient condition to get the node included in the graph, it shouldn't be making the too-broad assertion that it knows what all the tags are, which just happens to be right as an implementation detail in how you initially set up your test manifest.